activesupport/lib/active_support/core_ext/range/include_time_with_zone.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 require "active_support/time_with_zone"
+require "active_support/deprecation"
+
+ActiveSupport::Deprecation.warn(<<-MSG.squish)
```

Comment:
Before deprecating the file we need to deprecate the method itself. So instead of this patch we should should deprecations on line 23 and line 25 when the conditionals for `self.begin.is_a?(TimeWithZone)` and `self.end.is_a?(TimeWithZone)` are executed telling people to use `cover?` instead of `include?`

Reply:
@rafaelfranca I have updated the PR accordingly.